packages/react-composites/src/composites/ChatComposite/file-sharing/AttachmentUploadButton.tsx

```diff
@@ -79,6 +79,7 @@ export const AttachmentUploadButton = (props: AttachmentUploadButtonProps): JSX.
         </IconButton>
       </Stack>
       <input
+        data-ui-id="attachment-upload-button"
```

Comment:
If it's needed for tests, please use data-testid instead
Please see playwright and testing-library links for more details
https://playwright.dev/docs/locators#locate-by-test-id
https://testing-library.com/docs/queries/bytestid/